assets/i18n/en.toml

```diff
@@ -45,28 +45,28 @@ savingsTotalEnergy = "{total} kWh charged"
 tabTitle = "Live community"
 power = "Charging power"
 powerSub1 = "{activeClients} of {totalClients} participants"
-powerSub2 = "are charging right now"
+powerSub2 = "charging…"
 greenShare = "Solar share"
-greenShareSub1 = "power is covered by"
-greenShareSub2 = "PV & battery storage"
-greenEnergy = "Solar energy"
+greenShareSub1 = "power provided by"
+greenShareSub2 = "solar, and battery storage"
+greenEnergy = "Solar"
 greenEnergySub1 = "charged with evcc"
 greenEnergySub2 = "since October 2022"
 
 [footer.telemetry]
-optIn = "I want to contribute my data as well."
-optInMoreDetails = "More details are available {0}."
+optIn = "I want to contribute my data."
+optInMoreDetails = "More details at {0}."
 optInMoreDetailsLink = "here"
 optInSponsorship = "Sponsoring required."
 
 [footer.sponsor]
 titleSponsor = "You are a supporter"
 titleNoSponsor = "Support us"
-thanks = "Thanks for your support, {sponsor}! It helps us with the further development."
+thanks = "Thank you, {sponsor}! You have helped the development."
```

Comment:
```suggestion
thanks = "Thank you, {sponsor}. You have helped evcc."
```

Reply:
`You have helped `
Looks like a one-time-deal only.

Reply:
You‘re supporting the development of evcc.